server/jetstream_api.go

```diff
@@ -396,16 +396,18 @@ type JSApiStreamDeleteResponse struct {
 
 const JSApiStreamDeleteResponseType = "io.nats.jetstream.api.v1.stream_delete_response"
 
-// Maximum number of subject details we will send in the stream info.
-const JSMaxSubjectDetails = 100_000
```

Comment:
This was technically public so can't remove.

Reply:
Ok so renamed back to `JSMaxSubjectDetails`